.pre-commit-config.yaml

```
@@ -35,10 +35,11 @@ repos:
       - id: yamllint
 
   - repo: https://github.com/tier4/pre-commit-hooks-ros
-    rev: v0.4.0
+    rev: ros-include-guard
```

Released!
https://github.com/tier4/pre-commit-hooks-ros/releases/tag/v0.5.0

```suggestion
    rev: v0.5.0
```